src/node/rpc/node_frontend.h

```diff
@@ -354,6 +354,30 @@ namespace ccf
       make_read_only_endpoint("primary", HTTP_HEAD, is_primary)
         .set_forwarding_required(ForwardingRequired::Never)
         .install();
+
+      auto consensus_config = [this](CommandEndpointContext& args) {
+        // Query node for configurations, separate current from pending
+        if (consensus != nullptr)
+        {
+          auto cfg = consensus->get_latest_configuration();
+          nlohmann::json c;
+          for (auto& [nid, ninfo] : cfg)
+          {
+            nlohmann::json n;
+            n["address"] = fmt::format("{}:{}", ninfo.hostname, ninfo.port);
+            c[fmt::format("{}", nid)] = n;
+          }
+          args.rpc_ctx->set_response_body(c.dump());
+        }
+        else
+        {
+          args.rpc_ctx->set_response_status(HTTP_STATUS_NOT_FOUND);
```

Comment:
Perhaps add some error message here as well?

Reply:
Done, but to be honest this isn't overly likely to happen.